test/uk/gov/hmrc/oasdiscoveryproxy/controllers/OasDiscoveryControllerSpec.scala

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.identitymanagementserviceproxy.controller
+
+import com.github.tomakehurst.wiremock.client.WireMock.{status => _, _}
+import org.mockito.{ArgumentMatchers, MockitoSugar}
+import org.scalatest.OptionValues
+import org.scalatest.freespec.AsyncFreeSpec
+import org.scalatest.matchers.must.Matchers
+import play.api.http.ContentTypes
+import play.api.http.Status.OK
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.Helpers._
+import play.api.test.{FakeHeaders, FakeRequest}
+import play.api.{Application, Configuration}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.test.{HttpClientV2Support, WireMockSupport}
+import uk.gov.hmrc.oasdiscoveryproxy.service.AuthorizationDecorator
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+class OasDiscoveryControllerSpec extends AsyncFreeSpec
+  with Matchers
+  with WireMockSupport
+  with HttpClientV2Support
+  with OptionValues
+  with MockitoSugar {
+
+  "GET request" - {
+    "must be forwarded with headers" in {
+      val responseBody = """{"jam": "scones"}"""
+
+      stubFor(
+        get(urlEqualTo("/oas-discovery-stubs/v1/oas-deployments"))
```

Comment:
As above, this should have _internal_ in the path: `/oas-discovery-stubs/internal/v1/oas-deployments`

Reply:
No change required here, as discussed